AdyenCardScanner/Sources/ROIView.swift

```diff
@@ -6,7 +6,7 @@
 
 import UIKit
 
-class ROIView: UIView {
+internal class ROIView: UIView {
```

Comment:
Let's add a description above for the abbreviation or change the class name to `RegionOfInterestView`?

Reply:
Yes, good idea.

Reply:
Done!